app/assets/css/main.css

```
@@ -134,429 +135,407 @@
-    color: var(--primary-500);
-  }
-  .header span {
-    color: var(--accent-500);
-  }
+    /* If you need legacy class compatibility */
+    main.l-container {
+    } /* maps old <main> styles to .l-container */
+
+    /* Back-compat aliases (optional) */
+    .card {
+        composes: u-card;
```

Stylelint (17.3.0)

[error] Unexpected unknown property "composes" (property-no-unknown)

(property-no-unknown)